9.4/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -137,3 +139,18 @@ function set_passwords() {
 
   pg_ctl stop
 }
+
+# NOTE: https://github.com/openshift/postgresql/issues/76
```

Comment:
can you please replace the note with some description? you can still link the issue, but would be great to have something that does not force us to open the browser ;-)